### DISCUSSION TITLE
Add new Tentacle in Octobot

### DISCUSSION THREAD

**Discussion by piyushkummaar** [Drakkar-Software/OctoBot#1849] [Show and tell]
How I can add a new tentacle in the octobot.

**Comment by valouvaliavlo** [comment]
Hi @piyushkummaar,
You can find all details in the docs : https://developer.docs.octobot.online/coding-1/tentacle-package-development
You can also inspire you of what there is alread : https://github.com/Drakkar-Software/OctoBot-Tentacles
